django/api/models.py

```diff
@@ -119,3 +119,8 @@ class ApplicationInstance(models.Model):
     class Meta:
         permissions = (('has_dev_perm','Has dev permissions'),)
         unique_together = ('version_code', 'application',)
+
+class Feedback(models.Model):
+    date = models.DateTimeField(auto_now=True)
+    description = models.TextField(blank=True)
```

Comment:
We should only allow non-empty feedback

Reply:
But its working for me and its just a tweak so..

ACK

Reply:
I didn't want to add this logic in the frontend since it's only a tool for us developers